### PR TITLE
QUILLT-497 Adds ldclient-node

### PR DESCRIPTION
**Pull request by jdrake**
@ns-gbernsleone

### CODE REVIEW

File: nodePackages/@types/events/1.2.0.nix [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{ buildNodePackage, fetchUrlWithHeaders, namespaceTokens, nodePackages, pkgs }:
+buildNodePackage {
+    name = "events";
+    version = "1.2.0";
+    src = fetchUrlWithHeaders {
+      url = "https://registry.npmjs.org/@types/events/-/events-1.2.0.tgz";
+      sha1 = "81a6731ce4df43619e5c8c945383b3e62a89ea86";
+      headers.Authorization = "Bearer ${namespaceTokens.types}";
```
Comment (ns-gbernsleone) [inline nodePackages/@types/events/1.2.0.nix]: You're probably going to have to get rid of this and also patch in a `lib/index.js` on both `@types` things.

**Comment by jdrake** [comment]
Updated

**Review by ns-gbernsleone — APPROVED** [review]

**Comment by jdrake** [comment]
Updated with tunnel patch

**Comment by ns-gbernsleone** [comment]
LGTM

**Pull request merged**